utils.py

```diff
@@ -65,6 +88,50 @@ def get_layer_by_name(name):
     return None
 
 
+def add_or_update_ee_layer(eeObject, visParams, name, shown, opacity):
+    image = None
+
+    if not isinstance(eeObject, ee.Image) and not isinstance(eeObject, ee.FeatureCollection) and not isinstance(eeObject, ee.Feature) and not isinstance(eeObject, ee.Geometry):
+        err_str = "\n\nThe image argument in 'addLayer' function must be an instace of one of ee.Image, ee.Geometry, ee.Feature or ee.FeatureCollection."
+        raise AttributeError(err_str)
+
+    if isinstance(eeObject, ee.Geometry) or isinstance(eeObject, ee.Feature) or isinstance(eeObject, ee.FeatureCollection):
+        features = ee.FeatureCollection(eeObject)
+
+        width = 2
+
+        if 'width' in visParams:
+            width = visParams['width']
+
+        color = '000000'
+
+        if 'color' in visParams:
+            color = visParams['color']
+
+        image_fill = features.style(
+            **{'fillColor': color}).updateMask(ee.Image.constant(0.5))
```

Comment:
This line might benefit from running [`autopep8`](https://github.com/hhatto/autopep8).

Reply:
This is what autopep8 has generated